x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.tsx

```diff
@@ -361,7 +360,7 @@ export const ActionForm = ({
               messageVariables={messageVariables}
               actionGroups={actionGroups}
               defaultActionMessage={defaultActionMessage}
-              defaultParams={getDefaultActionParams?.(actionItem.actionTypeId, actionItem.group)}
+              defaultParams={getDefaultActionParams?.(actionItem.actionTypeId, actionItem.group, false)}
```

Comment:
why did we do this change?

Reply:
There were inconsistencies in the signature of that function